ReleaseProcessAutomation/Jira/JiraGetEarliestUnreleasedVersion.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
+using ReleaseProcessAutomation.Jira.ServiceFacadeInterfaces;
+
+namespace ReleaseProcessAutomation.Jira
+{
+  public class JiraGetEarliestUnreleasedVersion : JiraTask
```

Comment:
Since this class is not used I think we can just delete the dead code. Same goes for other parts which are not referenced. If somebody needs it in the future they can always use the history to find it. But I don't think it adds anything otherwise.

Or is there a future task where they are needed again?

cc @MichaelKetting

Reply:
no, unused stuff can be dropped.